examples/tensorflow/object_detection/README.md

```diff
@@ -157,22 +157,23 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
     --backbone-checkpoint=<path_to_resnet50-2018-02-07_folder>
 - Export trained model to the Keras H5 format.
 
-<a name='results'></a>
-
 ## Results
+<a name="results"></a>
 
-|**Model**|**Compression algorithm**|**Dataset**|**mAP (drop) %**|**NNCF config file**|**TensorFlow checkpoint**|
+|Model|Compression algorithm|Dataset|mAP (_drop_) %|NNCF config file|Checkpoint|
 | :---: | :---: | :---: | :---: | :---: | :---: |
-|RetinaNet|None|COCO2017|33.44|[retinanet_coco.json](configs/retinanet_coco.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/models/develop/tensorflow/retinanet.tar.gz)|
```

Comment:
Could you explain why you removed it?

Reply:
It was the result of a bug in the table-generating script - fixed the bug and the line reappeared.